<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexDialog</class>
 <widget class="QDialog" name="IndexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indexes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="ByRecordNumberButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>A..by Record Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ByTitleButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>B..Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ByAuthorLastButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>C..AuthorLast</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BySubjTitleButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>D..Subj+Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ByPubTitleButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>E..Pub+Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ByOrderActivButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>F..OrderActiv</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
